packages/clay-core/src/tree-view/useTree.ts

```diff
@@ -100,12 +110,21 @@ export function useTree<T>(props: ITreeProps<T>): ITreeState<T> {
 }
 
 // RFC 6902 4.4
-type Patch = {
+type PatchMove = {
 	op: 'move';
 	from: Array<number>;
 	path: Array<number>;
 };
 
+// RFC 6902 4.1
```

Comment:
Nothing wrong here, but I'm just wondering if those "RFC" related comments are going to cause confusion. I know what it means, but that's because I'm in the squad and have context about this, but I'm thinking of anyone using Clay (or simply in another squad).

Reply:
Yeah I add them to add references and create context, the idea is that people don't deviate from the implementation in the future or remember that this follows a pattern rather than just created by us, I can add more comments to also reduce confusion. What do you think?

Reply:
Well, looking at the code (once one is familiar with the component's API) it's clear what these patches do ... but I think adding a small comment would help yes, good idea, that works for me.

Reply:
Viewing this without context, I'm not sure what the RFC is referring to. Would it be possible to include a permalink to something here as well? Something like `// RFC 6902 https://github.com/liferay/clay/issues/10`

Reply:
Yeah, it's an option too.

Reply:
Thanks for the feedback @bryceosterhaus.

@matuzalemsteles the comments you added look good enough for me. Since our TreeView component is still in progress, I'll wait to get a "second round" of comments for Bryce - since he doesn't have context if this clarified his doubts I think we're all good.